src/poliastro/threebody/soi.py

```diff
@@ -0,0 +1,102 @@
+"""Sphere of Influence
+
+Contains methods to compute radius of sphere of influence.
+
+Laplace Radius: A laplace sphere of influence (SOI) in astrodynamics and astronomy is the oblate-spheroid-shaped
+region around a celestial body where the primary gravitational influence on an orbiting object is that body.
+This is usually used to describe the areas in the Solar System where planets dominate the orbits of
+surrounding objects such as moons, despite the presence of the much more massive but distant Sun. In the
+patched conic approximation, used in estimating the trajectories of bodies moving between the neighbourhoods
+of different masses using a two body approximation, ellipses and hyperbolae, the laplace radius is taken as
+the boundary where the trajectory switches which mass field it is influenced by.
+ The result is a * (m / M)^2/5.
```

Comment:
Good opportunity to use some LaTeX? :innocent:

Reply:
Right!